src/pages/verify/e-ids/finnish-trust-network.mdx

```diff
@@ -16,38 +16,267 @@ The requirements are described in <a href="https://www.kyberturvallisuuskeskus.f
 
 1. Private Key JWT client authentication
 2. Authentication request signing
-3. Encrypted token and userinfo responses
+3. Encrypted `token` and `userinfo` responses
 4. Statically configured JWK sets for signing and encryption
 
-Client applications must be able to fulfill the new requirements to continue offering FTN logins – regardless of the provider chosen for FTN integration.
+Client applications must be able to fulfill the new requirements to continue offering FTN logins, regardless of the chosen FTN broker. All changes must be implemented by August 31, 2025.
 
-As we're finalizing the technical changes on our side, you can start configuring your FTN integration to meet the new requirements. If you need any help or guidance, don't hesitate to reach out.
+### Technical guides
 
-**1. Private Key JWT client authentication**
+Technical guides for each requirement are provided below:
+
+- [Private Key JWT client authentication](/verify/e-ids/finnish-trust-network/#private-key-jwt-client-authentication)
+- [Authentication request signing](/verify/e-ids/finnish-trust-network/#authentication-request-signing)
+- [Encrypted token and userinfo responses](/verify/e-ids/finnish-trust-network/#encrypted-token-and-userinfo-responses)
+- [Statically configured JWK sets](/verify/e-ids/finnish-trust-network/#statically-configured-jwk-sets-for-signing-and-encryption)
+
+Should you encounter any challenges or need further assistance, please do not hesitate to reach out to our support team at [support@criipto.com](mailto:support@criipto.com) or via [Slack](https://tiny.cc/criipto-slack).
+
+### Test your implementation
+
+After implementing the changes described in the [technical guides](/verify/e-ids/finnish-trust-network/#technical-guides), you can test your setup by activating the **Enforce FTN requirements** toggle. You'll find it in your Criipto Dashboard under `eID providers` > `FTN`.
```

Comment:
There is also a special `enforce_ftn_requirements` login_hint that can be used to test specific requests.